Analysis/Utilities/RootScanner/source/HistScannerChanData.cpp

```diff
@@ -39,12 +39,10 @@ void HistScannerChanData::Set(XiaData *data) {
 
     //Do not delete as it causes segfault.
     ProcessedXiaData *chEvent = new ProcessedXiaData(*data);
-
-    //baseline[mod][chan] = chEvent->CorrectBaseline();
-
-    //peakAdc[mod][chan] = chEvent->maximum;
-    //traceQdc[mod][chan] = chEvent->IntegratePulse(chEvent->max_index - 10,
-    //                                              chEvent->max_index + 15);
+    //@TODO ensure that these work properly
+    baseline[mod][chan] = chEvent->GetTrace().GetBaselineInfo().first;
+    peakAdc[mod][chan] = chEvent->GetTrace().GetMaxInfo().second;
+    traceQdc[mod][chan] = chEvent->GetTrace().GetQdc();
```

Comment:
Can we open an issue and make sure these don't get forgotten?

Reply:
Never mind these are not what I thought they were (I thought it was cfd info).

Reply:
Sure you can!